### PR TITLE
feat: validations working for the Phone Input component

### PR DESCRIPTION
**Pull request by davidpaley**
Note: I should branched from staging but the reality is that I need this ASAP in master to have a new version of blueprint and can test everything with Portal. The change is really small and will not affect other part of the library.

### CODE REVIEW

File: lib/components/phoneInput.js [hunk 1]
```diff
@@ -19835,24 +19835,24 @@ function (_PureComponent) {
         placeholder: placeholder,
         value: value,
         onChange: this.handleChange,
-        error: isValidationOk() ? "" : getValidationMessage(),
-        className: className
+        error: isValidationOk() ? '' : getValidationMessage(),
+        className: "".concat(className, " ").concat(isValidationOk() && styles$1.error ? styles$1.error : 'PEPITOO')
```
Comment (rin) [inline lib/components/phoneInput.js]: PEPITOOOO!

File: src/lib/components/phoneInput/__snapshots__/phoneInput.test.js.snap [hunk 2]
```diff
@@ -5,7 +5,7 @@ exports[`PhoneInput renders properly 1`] = `
   className=" theme-purple inputWrapper"
 >
   <div
-    className="react-phone-number-input input"
+    className="react-phone-number-input input PEPITOO"
```
Comment (rin) [inline src/lib/components/phoneInput/__snapshots__/phoneInput.test.js.snap]: I am not entirely sure we should merge this to master :D .

File: lib/components/phoneInput.js (hunk 1 above)
Comment (kbardi) [reply lib/components/phoneInput.js]: haha, we shouldn't use a class PEPITOO

File: src/lib/components/phoneInput/__snapshots__/phoneInput.test.js.snap (hunk 2 above)
Comment (kbardi) [reply src/lib/components/phoneInput/__snapshots__/phoneInput.test.js.snap]: haha, interesting class

File: src/lib/components/phoneInput/phoneInput.js [hunk 3]
```diff
@@ -72,11 +72,14 @@ class PhoneInput extends PureComponent {
           placeholder={placeholder}
           value={value}
           onChange={this.handleChange}
-          error={isValidationOk() ? "" : getValidationMessage()}
-          className={className}
+          error={isValidationOk() ? '' : getValidationMessage()}
+          className={`${className} ${
+            isValidationOk() && styles.error ? styles.error : 'PEPITOO'
```
Comment (kbardi) [inline src/lib/components/phoneInput/phoneInput.js]: I think PEPITOO is for testing?

**Comment by davidpaley** [comment]
hahaha thanks @kbardi @rin xD updating this and then merging..

**Pull request merged**